src/global-table.ts

```diff
@@ -0,0 +1,34 @@
+import { IResource, Resource, aws_dynamodb, RemovalPolicy, Stack } from 'aws-cdk-lib';
+import { CfnGlobalTable } from 'aws-cdk-lib/aws-dynamodb';
```

Comment:
Import style needs to be fixed, see https://github.com/cdklabs/awscdk-dynamodb-global-tables/pull/1#discussion_r941431618